core/src/main/java/org/apache/accumulo/core/client/admin/compaction/CompactionConfigurer.java

```diff
@@ -47,10 +49,25 @@ public interface InitParameters {
   /**
    * @since 2.1.0
    */
-  public interface InputParameters {
+  interface InputParameters {
```

Comment:
The server code provides an implementation of these interfaces and then calls the methods on the CompactionConfigurer interface passing the object created on the server side. Is it the case that these interfaces (InitParameters and InputParameters) are *not* part of the public api and can change like this in a patch release?

Reply:
Ok, I'm good with this as long as we document it.

Reply:
I looked at the 2.1.4 draft release notes and did not see anything about semver violations, going to add a section for that.

Reply:
Reviewed all API and SPI changes in 2.1.4 and created https://github.com/apache/accumulo-website/pull/464.